pkg/service/service.go

```diff
@@ -14,6 +14,45 @@ type Health struct {
 func (s *Health) Push(context.Context, *proto.PushRequest) (*proto.PushResponse, error) {
 	return nil, errors.New("not implemented")
 }
-func (s *Health) Register(context.Context, *proto.RegisterRequest) (*proto.RegisterResponse, error) {
-	return nil, errors.New("not implemented")
+func (s Health) Register(c context.Context, reqproto.RegisterRequest) (proto.RegisterResponse, error) {
+    client, err := mongo.NewClient(options.Client().ApplyURI(mongoURI))
```

Comment:
client for mongodb should be created only once during Health service startup and kept as Health service parameter. so when you receive the request you can just use it